_styles/-theme.scss

```diff
@@ -0,0 +1,51 @@
+---
+---
+
+// colors
+[data-dark="false"] {
+  --primary: #6366f1;
```

Comment:
Although I made significant efforts to improve lighthouse scores in v1.1.0 and other upcoming changes (see branches), I often ignore this warning because it is often simply wrong. Here are some references:

https://twitter.com/DanHollick/status/1417895151003865090
https://uxmovement.com/buttons/the-myths-of-color-contrast-accessibility/
https://github.com/w3c/wcag/issues/695

In general, people take lighthouse and WCAG too much as gospel and objectively correct. Still, I will tweak the colors here a bit.